xml/System.Text.Unicode/Utf8.xml

```diff
@@ -41,15 +41,29 @@
         <Parameter Name="isFinalBlock" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="charsRead">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="replaceInvalidSequences">To be added.</param>
-        <param name="isFinalBlock">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">A UTF-16 encoded read-only character span.</param>
+        <param name="destination">A UTF-8 encoded byte span.</param>
+        <param name="charsRead">When the method returns, the number of characters read from <paramref name="source" />.</param>
+        <param name="bytesWritten">When the method returns, the number of characters written to <paramref name="destination" />.</param>
+        <param name="replaceInvalidSequences"><see langword="true" /> to replace invalid UTF-16 sequences in <paramref name="source" /> with U+FFFD; <see langword="false" /> to return <see cref="F:System.Buffers.OperationStatus.InvalidData" /> if invalid characters are found in <paramref name="source" />.</param>
+        <param name="isFinalBlock"><see langword="true" /> if the method should not return <see cref="F:System.Buffers.OperationStatus.InvalidData" />; otherwise, <see langword="false" />.</param>
+        <summary>Converts a UTF-16 character span to a UTF-8 encoded byte span.</summary>
+        <returns>A value that indicates that status of the conversion.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+### Remarks
+
+This method corresponds to the [UTFEncoding.GetBytes](xref:System.Text.UTF8Encoding.GetBytes%2A) method, except that it has a different calling convention, different error handling mechanisms, and different performance characteristics. 
```

Comment:
The formatting used for this GetBytes xref is unusual. @rpetrusha can you explain why you added it that way?

Reply:
I hate to admit it, @carlossanlop, but I can't think of why I might have done that. It allows you to link to customize the text used to describe a type/member or assign a type/member name other than the one you're linking to (such as when you want to describe an inherited member as if it belong to the derived class), but I can't think of why I used it here. It build successfully, in any case.

Reply:
As long as it compiles and looks good, it's fine.